com.cocoatech.deckr.yaml

```diff
@@ -0,0 +1,129 @@
+app-id: com.cocoatech.deckr
+runtime: org.gnome.Platform
+runtime-version: '43'
```

Comment:
44 is out

Reply:
but do you really need the GNOME runtime?

Reply:
I don't know.  Don't I need that to display a window?  Why would I not need it?

Reply:
error: org.gnome.Sdk/x86_64/44 not installed
Failed to init: Unable to find sdk org.gnome.Sdk version 44

$ flatpak update
Looking for updates…
F: Warning: Treating remote fetch error as non-fatal since app/com.example.FlutterApp/x86_64/master is already installed: No such ref 'app/com.example.FlutterApp/x86_64/master' in remote flutterapp-origin
Nothing to do.

I searched the docs, I have no idea why 44 doesn't work.  very confusing.

Reply:
I tried freedesktop and it failed

-- Checking for modules 'glib-2.0>=2.58;gtk+-3.0>=3.24'
--   Found glib-2.0, version 2.72.4
--   Found gtk+-3.0, version 3.24.36
-- Checking for module 'gobject-introspection-1.0'
--   Found gobject-introspection-1.0, version 1.72.1
CMake Error at /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Vala (missing: VALA_COMPILER)
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindVala.cmake:52 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  src/CMakeLists.txt:164 (find_package)